cogs/fun.py

```diff
@@ -326,6 +326,15 @@ async def xkcdcomic(self, ctx):
 
         await ctx.send(f"https://xkcd.com/" + str(random.randint(1,2400)) + "/")
 
+    @commands.command(aliases=['guess'])
+    async def guess_the_game(self, ctx, player_guess: int):
+        """Guess a number between 1-10"""
+        computer_number = random.randint(1, 11)
```

Comment:
`random.randint()`, unlike the usual in Python, includes both endpoints. So this will generate a random number from 1 to *11*, not 1 to 10. Easy fix:
```suggestion
        """Guess a number between 1-10"""
        computer_number = random.randint(1, 10)
```